modules/ROOT/pages/platform/security.adoc

```diff
@@ -179,28 +179,43 @@ Please see the https://learn.microsoft.com/en-us/azure/private-link/rbac-permiss
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-AuraDB Enterprise supports Single Sign-On (SSO) for accessing the Bloom and Browser clients.
+Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
 
-The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported:
+[NOTE]
+====
+Aura requires Authorization Code Flow with PKCE to ensure best practice security.
+====
+
+=== Console SSO
+
+Console SSO allows users to log in to the Aura Console using their company IdP credentials and grants link:{neo4j-docs-base-uri}/cypher-manual/current/administration/access-control/built-in-roles#access-control-built-in-roles-public[Public Access privileges] to all instances in the tenant.
+
+The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported for Console-level Authentication:
 
 * Microsoft Azure Active Directory (AAD)
 * Okta
-* Keycloak
-* Google Authentication
 
-[NOTE]
-====
-If the IdP you are currently using is not in our list of certified IdPs above, please let us know using the support ticket method mentioned below and we will evaluate the possibility of adding support.
-====
+To enable Console SSO on your Aura Enterprise tenant(s), please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
+
+. The _Tenant ID_ of the tenant(s) you want to use SSO. See xref:platform/user-management.adoc#_tenants[Tenants] for more information on how to find your __Tenant ID__.
+. The name of your IdP.
+
+=== Instance SSO
+
+Instance SSO allows you to directly map groups of users (as defined in your IdP) to DBMS RBAC roles when launching Workspace, Bloom and Browser clients from an Aura instance.
 
-Aura supports Authorization Code Flow with PKCE to ensure best practice security.
+The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported for instance-level Authentication:
+
+* Microsoft Azure Active Directory (AAD)
+* Okta
+* Keycloak
+* Google Authentication
 
-To add SSO for Browser and Bloom to your AuraDB Enterprise instances, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
+To add SSO for Workspace, Bloom and Browser to your Aura Enterprise instances, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
```

Comment:
Oxford comma here? We do recommend it in our style guide

Reply:
```suggestion
To add SSO for Workspace, Bloom, and Browser to your Aura Enterprise instances, please https://aura.support.neo4j.com/hc/en-us/requests/new[raise a support ticket] including the following information:
```